core/src/components/radio-group/test/form/radio-group.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('radio-group: form', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/src/components/radio-group/test/form');
+  });
+
+  test('selecting an option should update the value', async ({ page }) => {
+    const radioGroup = page.locator('ion-radio-group');
+    const ionChange = await page.spyOnEvent('ionChange');
+    const griffRadio = page.locator('ion-radio[value="griff"]');
+
+    await expect(radioGroup).toHaveAttribute('value', 'biff');
+
+    await griffRadio.click();
+    await page.waitForChanges();
+
+    await expect(ionChange).toHaveReceivedEventDetail({ value: 'griff' });
+  });
+
+  test('selecting a disabled option should not update the value', async ({ page }) => {
```

Comment:
May want to use `toHaveAttribute`/`toHaveReceivedEventDetail` on this test too? Otherwise this LGTM 👍

Reply:
The reason I opted against this for this test case is:
1. When the radio is disabled, `ionChange` will not be emitted, so we cannot check for the spy to have received an event.
2. When the value is changed, the reflected value on the `ion-radio-group` is not updated. Checking `toHaveAttribute` would capture no change from interaction. 

I am unsure if 2 is a bug or not. It currently is the same behavior as `ion-input` (the reflected value doesn't match user interaction unless set manually/from a framework) & consistent with how the built-in input works.